string/string.mbt

```diff
@@ -201,3 +201,35 @@ test "chars" {
     ,
   )?
 }
+
+pub fn starts_with(self : String, str : String) -> Bool {
+  if str.length() > self.length() {
+    false
+  } else {
+    self.substring(end=str.length()).compare(str) == 0
+  }
+}
+
+test "starts_with" {
+  @assertion.assert_eq(true, "abc".starts_with("a"))?
+  @assertion.assert_eq(true, "abc".starts_with("ab"))?
```

Comment:
We have `assert_false` and `assert_true`.

Reply:
👌🏻